src/Controllers/QuizController.php

```diff
@@ -46,20 +46,39 @@ public function setQuestionService(AbstractService $questionService)
         $this->questionService = $questionService;
     }
 
+    /**
+     * Gets all the entities and displays them on them either on
+     * admin quizzes page or on user quizzes page
```

Comment:
I will fix this on the next commit

Reply:
This comment refers to an issue with the language used in the comments. Ok.